KinectIPD/KinectIPD/EncDec.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace KinectIPD
+{
+    public class EncDec
+    {
+        private static Encoding _encoding = Encoding.Default;
+        public static byte[] Protect(string stringData)
+        {            
+            return Protect(_encoding.GetBytes(stringData));
+        }
+        public static byte[] Protect(byte[] data)
+        {
+            try
+            {
+                // Encrypt the data using DataProtectionScope.CurrentUser. The result can be decrypted
+                //  only by the same current user.
+                return ProtectedData.Protect(data, null, DataProtectionScope.CurrentUser);
+            }
+            catch (CryptographicException e)
+            {
+                Console.WriteLine("Data was not encrypted. An error occurred.");
```

Comment:
NIT: I don't know the system well enough but I wonder if it should produce a log file for ease of use.